packages/lesswrong/lib/collections/posts/schema.ts

```diff
@@ -542,12 +542,14 @@ const schema = {
     type: "Comment",
     graphQLtype: "Comment",
     viewableBy: ['guests'],
-    resolver: async (post) => {
+    resolver: async (post, args, { currentUser, TagRels }) => {
       if (post.question) {
         if (post.lastCommentPromotedAt) {
-          return Comments.findOne({postId: post._id, answer: true, promoted: true}, {sort:{promotedAt: -1}})
+          const comment = await Comments.findOne({postId: post._id, answer: true, promoted: true}, {sort:{promotedAt: -1}})
+          return accessFilterSingle(currentUser, TagRels, comment)
```

Comment:
why is this using TagRels instead of Comments?

Reply:
Oops, sorry. That was an accident.

Reply:
(Fixed)

Reply:
(Fixed)